code/API_definitions/NetworkSliceBooking.yaml

```diff
@@ -0,0 +1,430 @@
+openapi: 3.0.3
+info:
+  title: API for Network Slice Booking
+  description: |
+    The Network Slice Booking (NSB) API provides programmable interface for developers to reserve a slice resource of a selected area within a period, and manage device access control as needed.
+  version: wip
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+servers:
+  - url: "{apiRoot}/nsb/vwip"
+    variables:
+      apiRoot:
+        default: http://localhost:9100
+tags:
+  - name: NSB Sessions
+    description: Manage NSB sessions
+paths:
+  /sessions:
+    post:
+      tags:
+        - NSB Sessions
+      summary: Creates a new session
+      operationId: createSession
+      requestBody:
+        description: Parameters to create a new session
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateSession"
+        required: true
+
+      responses:
+        "201":
+          description: Session created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SessionInfo"
+        "400":
+          description: Invalid input for createSession operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorInfo"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "503":
+          $ref: "#/components/responses/Generic503"
+
+  /sessions/{sessionId}:
+    delete:
+      tags:
+        - NSB Sessions
+      summary: Delete a NSB session
+      description: Deleting NSB session
+      operationId: deleteSession
+      parameters:
+        - name: sessionId
+          in: path
+          description: Session ID that was obtained from the createSession operation
+          required: true
+          schema:
+            $ref: "#/components/schemas/SessionId"
+      responses:
+        "204":
+          description: Session deleted
+        "400":
+          description: Invalid input for createSession operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorInfo"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "503":
+          $ref: "#/components/responses/Generic503"
+    get:
+      tags:
+        - NSB Sessions

```

Comment:
```suggestion
        - Network Slice Booking Sessions
```

Reply:
@ShutingQing you need to commit this suggestion as well, otherwise it doesn't fit to line 17